src/js/vre/chapter31/labels.js

```diff
@@ -0,0 +1,8 @@
+export const dischargeTypeLabels = {
```

Comment:
We could move these up a folder and use them in both forms, probably.

Reply:
Would it make sense to follow the edu form structure and use a utils folder to hold labels, etc?